### PR TITLE
Updated snackbar

### PR DESCRIPTION
**Pull request by marcelinkaaa**
Just an updated look of the notification. If you guys like it. 

screenshot_20160613-211629_1024

### CODE REVIEW

File: android/res/layout/view_rich_notification.xml [hunk 1]
```diff
@@ -1,5 +1,4 @@
-<?xml version="1.0" encoding="utf-8"?>
-<!--
+<?xml version="1.0" encoding="utf-8"?><!--
```
Comment (aldenml) [inline android/res/layout/view_rich_notification.xml]: do you mind putting the `<!--` in a new line

Comment (marcelinkaaa) [inline android/res/layout/view_rich_notification.xml]: sure in a sec. :)

File: android/res/layout/view_rich_notification.xml [hunk 2]
```diff
@@ -20,73 +19,71 @@
...
+        style="@style/RichNotificationButtonText"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
         android:layout_gravity="center_vertical|center_horizontal"
-        android:background="@null"
-        android:layout_marginRight="5sp"
-        android:src="@drawable/clearable_edittext_clear_icon"/>
-</LinearLayout>
\ No newline at end of file
+        android:text="@string/ok" />
```
Comment (aldenml) [inline android/res/layout/view_rich_notification.xml]: You  can use `@android:string/ok` instead and delete the new entry in ours `string.xml`

**Comment by aldenml** [comment]
See my last comment and I'm ready to merge this

**Pull request merged**